packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -112,9 +117,21 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
 
         this.state = {
             isValid: true,
+            containers: undefined,
         };
     }
 
+    componentDidMount() {
```

Comment:
Unrelated to your changes, but would you mind flipping the `UNSAFE_componentWillMount` in the `AssayPropertiesInput.tsx` case to `componentDidMount`. I think for this usage they should be functionally equivalient.

Reply:
👍